fix(NcActionInput): Fix multi select in NcActionInput

nickvergessen opened the pull request:
Before | After
---|---
Scrolling is a nightmare | Scrolling works reasonably well
![grafik](https://user-images.githubusercontent.com/213943/220105674-88595727-4688-46ae-92f3-e63c63753a97.png) | ![grafik](https://user-images.githubusercontent.com/213943/220107047-d2e45e4f-7004-4f78-9532-86eef0649b97.png)


The visuals are still broken, but at least it's not `[object Object]` anymore.

Can someone help with the layering?

raimund-schluessler commented:
You guys don't like https://github.com/nextcloud/nextcloud-vue/pull/3760 huh? 🙈

nickvergessen commented:
> You guys don't like https://github.com/nextcloud/nextcloud-vue/pull/3760 huh? :see_no_evil:

Sorry, I just updated to the latest version, got the [bug reported](https://github.com/nextcloud/announcementcenter/issues/607) went to the lib and fixed it.
Would not have thought that there is a pending PR for a bug like this.

nickvergessen commented:
We can cherry-pick my documentation part to yours and merge that?

raimund-schluessler commented:
> We can cherry-pick my documentation part to yours and merge that?

Fine with me. 👍

nickvergessen commented:
I will send a commit to your PR to add the sample
